### DISCUSSION TITLE
Chatops hubot slack adapter can't connect.

### DISCUSSION THREAD

**Discussion by gableb** [StackStorm/st2#5501] [ChatOps]
st2 version: st2 3.6.0, on Python 3.8.10

OS version: Ubuntu 20.04

Python version: Python 3.8.10

Journalctl output upon startup:
https://gist.github.com/gableb/da31c3d97d2b9795ae21259676759a42

When starting the st2chatops service, the following logs appear. The hubot slack adapter does not connect to our slack instance.

**Comment by nzlosh** [comment]
The workaround is posted here https://github.com/slackapi/hubot-slack/issues/636 or you can use an alternative bot like https://err-stackstorm.readthedocs.io/en/latest/getting_started.html#overview

**Comment by amanda11** [comment]
I've pushed a fix for this in for ST2 3.7.0 - https://github.com/StackStorm/st2chatops/pull/173. Just need to add the variable mentioned above into your st2chatops.env.
